src/static/css/homepage.css

```diff
@@ -0,0 +1,26 @@
+.asd {
+    padding: 60px;
+    text-align: center;
+    background: #6C7A89;
+    color: white;
+}  
+
+.asd > .title {
+    font-size: 50px;
```

Comment:
consider more responsive font sizes; https://kyleschaeffer.com/css-font-size-em-vs-px-vs-pt-vs-percent, https://stackoverflow.com/questions/609517/why-em-instead-of-px

Reply:
switched to em